docs/projects/cn0577/index.rst

```diff
@@ -0,0 +1,294 @@
+   :widths: 20 20 20 20 20
+   :header-rows: 1
+
+   * - I2C type
+     - I2C manager instance
+     - Alias
+     - Address
+     - I2C subordinate
+   * - PL
+     - iic_fmc
+     - axi_iic_fmc
+     - 0x4162_0000
+     - ---
+   * - PL
+     - iic_main
+     - axi_iic_main
+     - 0x4160_0000
+     - ---
+
+GPIOs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 25 20 20 20 15
+   :header-rows: 2
+
+   * - GPIO signal
+     - Direction
+     - HDL GPIO EMIO
+     - Software GPIO
+     - Software GPIO
+   * -
+     - (from FPGA view)
+     -
+     - Zynq-7000
+     - Zynq MP
+   * - pd_cntrl
+     - IN
+     - 32
+     - 86
+     - 110
+   * - testpat_cntrl
+     - IN
+     - 33
+     - 87
+     - 111
+
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+================ === ========== ===========
+Instance name    HDL Linux Zynq Actual Zynq
+================ === ========== ===========
+ad_cpu_interrupt 13  57         89
+================ === ========== ===========
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here <>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+.. code-block::
+   :linenos:
+
+   user@analog:~$ cd hdl/projects/cn0577/zed
+   user@analog:~/hdl/projects/cn0577/zed$ make
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user
+guide.
+
+Resources
+-------------------------------------------------------------------------------
+
+Systems related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :dokuwiki:`[Wiki] EVAL-CN0577-FMCZ User Guide <resources/eval/user-guides/circuits-from-the-lab/cn0577>`
+-  :dokuwiki:`[Wiki] CN0577 HDL Reference Design <resources/eval/user-guides/circuits-from-the-lab/cn0577/hdl>`
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Product datasheets:
+
+   -  :adi:`ADAQ23876`
+   -  :adi:`ADR4520`
+   -  :adi:`ADA4945-1`
+   -  :adi:`ADN4661`
+-  `Circuit Note CN-0577 <https://www.analog.com/media/en/reference-design-documentation/reference-designs/cn0577.pdf>`__
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`CN0577 HDL project source code <projects/cn0577>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - AXI_LTC2387
+     - :git-hdl:`library/axi_ltc2387`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_ltc2387>`
```

Comment:
All the links from dokuwiki should be changed to :ref: because we already have them ported

Reply:
Doing this the links won't work anymore. I did change only for AXI_DMAC and AXI_PWM_GEN. If I change the other links, they won't work. Is there a new update to the tags?

Reply:
Changed the labels to :ref: of the items that are already ported.